fix: commissions screen fixes

liordino opened the pull request:
fix: don't allow negative values on the commissions forms

feat: add an all machines option to the commissions overrides

fix: added cash-in and cash-out icons to main and override table headers

feat: filter out already overriden machine-coin combinations

fix: fix error when there's no machine selected yet

liordino committed:
fix: don't allow negative values on the commissions forms

feat: add an all machines option to the commissions overrides

fix: added cash-in and cash-out icons to main and override table headers

feat: filter out already overriden machine-coin combinations

fix: fix error when there's no machine selected yet

liordino commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Commissions screen fixes](https://trello.com/c/zjR0cCy2/173-commissions-screen-fixes)

RafaelTaranto commented on new-lamassu-admin/src/pages/Commissions/helper.js:
```diff
@@ -1,10 +1,35 @@
+const ALL_MACHINES = {
+  name: 'All Machines',
+  deviceId: 'ALL_MACHINES',
+  __typename: 'Machine'
```
I don't think we need __typename here

RafaelTaranto commented on new-lamassu-admin/src/pages/Commissions/helper.js:
```diff
@@ -188,8 +246,8 @@ const defaults = {
-  machine: '',
-  cryptoCurrencies: [],
+  machine: null,
+  cryptoCurrencies: '',
```
Why did you need to change from empty array to ''? Isn't cryptoCurrencies a select multiple?

liordino committed:
fix: removed unnecessary __typename property on 'All machines' option

fix: cryptoCurrencies is a select multiple so it's default should be [] instead of ''
